### ISSUE TITLE
# 3 Accessibility

### ISSUE THREAD

**Issue by LDeLaRosa13** [LDeLaRosa13/The-Year-To-Live#8]
-You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.
